config/install/metatag.metatag_defaults.global.yml

```diff
@@ -13,4 +13,4 @@ tags:
   og_type: article
   og_image: '[site:url]/profiles/cr/themes/custom/campaign_base/images/og/rnd-logo_open-graph_v2.jpg'
   og_url: '[current-page:url:absolute]'
-  bing: null
+  og_locale: en_GB
```

Comment:
@Saphyel how would this work for non-UK sites? Thinking e.g. about RND USA, or others that are in the pipeline?

Reply:
well this is a recommendation, they can update the value in their site config.